packages/ocamlog/ocamlog.0.0.1/opam

```diff
@@ -0,0 +1,22 @@
+
+opam-version: "2.0"
+synopsis: "Simple Logger for OCaml"
+description: "Able to print with different colors, levels, trace out caller, etc."
+maintainer: "paulpatault <p.patault@gmail.com>"
+authors: "paulpatault <p.patault@gmail.com>"
+depends: [
+  "ocaml"
+  "dune" {>= "2.8"}
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+dev-repo: "git+https://github.com/paulpatault/OcamLog"
+homepage: "https://www.github.com/paulpatault/OcamLog"
+bug-reports: "https://www.github.com/paulpatault/OcamLog/issues"
+license: "MIT"
+url {
+  src: "https://github.com/paulpatault/ocamlog/archive/v0.0.1.tar.gz"
```

Comment:
The `v0.0.1` tag hasn’t changed for a few days and thus the changes you’re made is still not in this release. Could you retag and force-push it?